mldsa/sign.c

```diff
@@ -143,71 +144,80 @@ int crypto_sign_signature_internal(uint8_t *sig, size_t *siglen,
   polyveck_ntt(&s2);
   polyveck_ntt(&t0);
 
-rej:
-  /* Sample intermediate vector y */
-  polyvecl_uniform_gamma1(&y, rhoprime, nonce++);
+  /* Reference: This code is re-structured using a while(true), */
+  /* with explicit "continue" statements rather than the "goto" */
+  /* so there is a syntactic location to place the loop         */
+  /* invariants for CBMC.                                       */
```

Comment:
nit: clarify that "continue" happens when a signature is rejected.

Reply:
Done